src/chart/pie/PieSeries.ts

```diff
@@ -122,12 +122,11 @@ class PieSeriesModel extends SeriesModel<PieSeriesOption> {
 
     static type = 'series.pie' as const;
 
-    useColorPaletteOnData = true;
-
     /**
      * @overwrite
      */
     init(option: PieSeriesOption): void {
+        console.log('init');
```

Comment:
I assume this is a debug print. Does echarts build remove console log automatically?

Reply:
Thanks for correcting. That should be removed.
If you want to log information in dev mode, `if (__DEV__)` should be used.